addons/web/static/src/legacy/xml/web_calendar.xml

```diff
@@ -151,7 +151,8 @@
                     <a href="#"
                         t-attf-class="o_cw_popover_link o_calendar_color_#{typeof color === 'number' ? _.str.sprintf('%s', color) : '1'} o_attendee_status_#{record.is_alone ? 'alone' : record.attendee_status} btn-link d-block"
                         t-att-data-id="record.id"
-                        t-att-data-title="record.display_name">
+                        t-att-data-title="record.display_name"
+                        t-att-data-state="record.state" >
```

Comment:
Looking into doing this via inheritance...

Reply:
Did you try with a `t-extend`?

```xml
<t t-extend="CalendarView.yearEvent.popover">
    <t t-jquery="blabla" position="attributes">
        <!-- your stuff here -->
    </t>
</t>
```

Reply:
Thanks :) I have added a second commit with the view inheritance but it seems that the <a> tag with a o_cw_popover_link class is not found :( I keep looking into it :)